### ISSUE TITLE
Fix Onan Generator

### ISSUE THREAD

**Issue by SimplGy** [SimplGy/itasca-navion-23J#58]
Starts, then shuts off.

Used the manual to determine fault code: 14
- [x] How can I read the frequency of the generator?
- [x] Does an LP generator have a carburator? -- Yes it does, the altitude adjustment knob is attached to it.

> Fault Code 14
> OverFrequency fault. Engine governor unable to maintain rated frequency

![image](https://cloud.githubusercontent.com/assets/548809/16400277/25443794-3c97-11e6-8117-3a4fe2c94d65.png)

**Comment by SimplGy** [comment]
https://www.google.com/search?q=onan+3.6kyfa26120L&gws_rd=ssl#q=onan+3600+code+14

http://www.rvforum.net/SMF_forum/index.php?topic=16528.0

> that's another symptom of a malfunctioning computer control board.
> 
> It sounds to me like a governor error,  There is a lever with a spring on it, IIRC this lever either has a serrated edge (The spring nestles in one of the serrations) or many holes (Spring hooks in a hole) and the spring may need to be moved to a different spot on the lever.  I'm not sure if it needs to be moved out or in.
> 
> Turns out I needed a new carb!  The service manager said I probably didn't run it frequently enough.

http://www.rv-coach.com/current_category.181/Forum.91195/offset.570/forum_thread_full.html

> got the rig back from Rocky Mtn Cummins, and after having to call Cummins directly... they replaced the carb (they did not find any varnish), and only explanation they could give was a sticking overspeed throttle controller.
> 
> Fixing the issue: you must clean/replace the carb. this can be a difficult task to clean. Most of the mechanics go with replace it, because it will cost you more for the labor to clean and then reinstall. 
> 
> Keeping the issue from happening again:
> you should run your generator, (regardless of how often you use your RV) at a minimum once a month for two hours, with half the load on it. This is called excercising your electrical. If you do this, you will never have any difficulty with your generator again.
> 
> You should try and find out what frequency the generator is running at. It should be around 60hz. A good multimeter should have this function. You should be able to read the voltage and Hertz also.
> 
>  Things to check include the two nuts holding the carb air cleaner on, the bolts in the intake manifold, & the small vaccum hose leading to the choke pull off diaphram as well as the diaphram itself. Also make sure the crankcase breather is clean & working properly.

http://forums.goodsamclub.com/index.cfm/fuseaction/thread/tid/27409957/srt/pd/pging/1.cfm

http://www.rvmechanic.com/current_category.181/Forum.112922/forum_thread.html

http://www.justanswer.com/small-engine/6oh8g-onan-genset-won-t-stay-running-gives-code-14-overfrequency.html

> Code 14, Over Frequency, means that the engine is running too fast.
> This can have several causes - air leak at the intake, carb and linkages not properly installed, misadjusted governor, broken governor.
> 
> Connect a frequency meter to the AC output leads of the genset, and check the frequency for the second the engine will run.
> What is the frequency?
> 
> On some of these carbs, it is possible (and very easy) to get the choke linkage connected backwards.
> I have uploaded a digram of what the linkage should look like. Note how the choke tang is pointed toward the rear of the set, away from you, and not toward the front of the set.
> If yours is backwards, you need to spin it around.
> 
> if the set will stay running at 59 Hz, but jumps to 70 Hz on it's own, then all you need to do is adjust the governor.

http://www.rv.net/forum/index.cfm/fuseaction/thread/tid/20700075.cfm

**Comment by SimplGy** [comment]
http://www.doityourself.com/forum/pick-up-trucks-large-passenger-cargo-vans-campers-trailers-rvs-motor-homes/362489-rv-generator-frequency-question.html#b

> Have you viewed the output waveform on an oscilloscope? Is it possible to get a schematic of the inverter section? If so, there has to be some sort of frequency-adjust control somewhere, even if it is microprocessor-controlled. Look for a board-mounted potentiometer. If you can view the output waveform, or have a frequency counter, I would recommend adjusting it until the frequency comes in line.
> If your inverter has a true sinewave output, to achieve a rather high degree of efficiency, the DC-AC inverter's frequency is usually 20-60kHz, modulated at 60Hz. This is then run thru a big L-C ilter to remove the high frequency component, leaving only the clean, 60Hz sinewave.
> If you're uncomfortable with working with high AC & DC voltages, take the unit to an authorised service center.

and...

> with the Inverter generators there are few diffrent type.
> Here are few most common types
> • Switched types { they have fair regulation but not very good sine wave format. ( useally flat sine wave format ) }
> • Modifed Sine wave { This is a cross between the switched and true sine wave format. This is the most common used inverter system and they do have good voltage / Hz regulations }
> • True Sine wave { this is a power company grade format and it is very tight control with voltage and HZ and it is very clean source useally don't have any spike something the lower graded products do }
> Anyway senice you got good voltage reading but HZ reading is bad that mean the control card is not getting proper singal or poor connection or interfernace with it.
> Try to put a load on like a heater or table lamp { no electronic control first on this test } and read both voltage and HZ to make sure it is true.
> If still haveing issue you may want to take this to the athrozied dealership or service centre to get this inverter / voltage regulator controller repace and the way I did got in the wiring digaram and this is all electronic and this type of inverter generator that something I don't see that small beside one manufacter I know they do make them as well.

**Comment by SimplGy** [comment]
[how to measure frequency](http://en-us.fluke.com/training/training-library/test-tools/digital-multimeters/how-to-measure-frequency-with-a-digital-multimeter.html)

**Comment by SimplGy** [comment]
https://www.youtube.com/watch?v=ZNNnxbaVt9M
![image](https://cloud.githubusercontent.com/assets/548809/16463057/5dfa4ffa-3df1-11e6-80c3-f97cc882ca38.png)
![image](https://cloud.githubusercontent.com/assets/548809/16463112/8e69a1e0-3df1-11e6-84be-a95a04057420.png)

**Comment by SimplGy** [comment]
https://www.youtube.com/watch?v=RfDrdRvDo-o

![image](https://cloud.githubusercontent.com/assets/548809/16463313/86c90f10-3df2-11e6-8998-9d59dac9c632.png)

**Comment by SimplGy** [comment]
http://www.trailerlife.com/trailer-how-to/trailer-tech/watts-up/

![image](https://cloud.githubusercontent.com/assets/548809/16463378/d2c47daa-3df2-11e6-9c12-17c6e6ea4305.png)

> The most common cause of hard starting (or not starting at all) with an RV generator is lack of use. Smith Powerhouse recommends that you run your generator every four weeks for two hours under load (such as a running air conditioner) to keep it properly exercised. The discoloration at the bottom of this carburetor float bowl means that fuel sat there for several months, and the deposits clogged the tiny orifices in the carburetor. Once allowed to deteriorate to this point, you’re looking at a $300 bill for the carburetor, plus removal/replacement labor. If running the generator regularly isn’t an option, Smith Powerhouse recommends installing a shutoff valve in the fuel line before the generator. Turn the valve off, then run the generator until it is out of fuel. Adding a fuel stabilizer (like Onan OnaFresh) is also recommended.

![image](https://cloud.githubusercontent.com/assets/548809/16463435/0fb29d0a-3df3-11e6-9c7a-a630ccd1519e.png)

> Wherever you take your generator for service or troubleshooting, Smith Powerhouse recommends you ask the shop personnel if they have a load bank. Similar to a dynamometer for cars, a load bank can load the generator to different percentages to make sure it’s running properly and producing the correct frequency, amperage and voltage for a given load. Smith Powerhouse conducts this test with every generator it services.

**Comment by SimplGy** [comment]
```
x Call Onan technicians re: generator https://github.com/SimplGy/itasca-navion-23J/issues/58
    x Model: Microquiet 3600 LP (3.6kyfa26120L)
    x Does an LP generator have a carburator? -- yes, definitely
    How can I read the frequency of the generator?
    x (303) 287-0201 Cummins mfg -- booked 'till July, seemed happy to offer phone assistance
    \ (303) 322-6013 B&B RV -- left message for Mark with Theresa -- They don't work on LP. Mark says to go to dealer
  \ (303) 756-9494 RV accessories -- left vm -- she called back, no genny work
    \ (303) 329-3006 -- no generator service
    \ (303) 422-7304 -- doesn't work on LP gennys
    \ (303) 216-2003 four seasons -- only filter and oil changes on gennys
    \ (303) 288-5361 -- no generator service
    \ (303) 478-4966 -- left VM asking for a call if they work on LP gennys -- he left vm, no genny work
    x (303) 287-3414 Spitzer -- Ray can work on it. Busy for a week or more though. He was awesome. says try spraying carb cleaner in the air filter, or take the carb out and clean it. need carb cleaner and compressed air.
    x (303) 623-8658 Carl does genny service, left vm
    \ (303) 429-6114 Nolans -- no lp genny service
    x (303) 289-3161 Transwest -- Talked to mobile dispatch, she's asking techs if they'll work on it.
```

**Comment by SimplGy** [comment]
grahammotorservice.com in Traverse City, MI was happy to look at it. Blake found the issue immediately and fixed it with me in the parking lot, teaching me a little as we went.

Turns out the blower hose (which is air intake for the regulator) was plugged. He cleaned it out with a wire and now it starts easier, runs smoothly, and doesn't shut off.

It makes sense that restricted airflow would cause a rich mixture and the generator running too fast. Very pleased with this outcome.
